community/php7/php7-fpm.logrotate

```diff
@@ -8,6 +8,6 @@
 	dateext
 	sharedscripts
 	postrotate
-		/etc/init.d/php-fpm7 --ifstarted --quiet reopen
+		/etc/init.d/php7-fpm --ifstarted --quiet reopen
```

Comment:
Firstly I remove suffixes at all. But then to implement coexistance with php5 I make suffixed 'php' part of name. 

It's mostly for install-scripts and trigger